README.md

```diff
@@ -93,10 +122,38 @@ An example extra specs json would look like this:
     "network_id": "542b68dd-4b3d-459d-8531-34d5e779d4d6",
     "storage_backend": "cinder_nvme",
     "boot_disk_size": 150,
-    "use_config_drive": false
+    "use_config_drive": false,
+    "disable_updates": true,
+    "enable_boot_debug": true,
+    "extra_context": {
+    "GolangDownloadURL": "https://go.dev/dl/go1.22.4.linux-amd64.tar.gz"
```

Comment:
Please indent. It's difficult to distinguish to which map/array the elements belong to.